modeller.py

```diff
@@ -0,0 +1,87 @@
+import math
+from pyspark.mllib.recommendation import ALS
+from operator import itemgetter
+import itertools
+import time
+import numpy as np
+
+
+class Estimator:
+    def __init__(self, data):
+        self._data = data
+        self._sets = self._split([0.63212056, 0.1839397, 0.1839397])
```

Comment:
should probably add a comment about these numbers, like why did we choose them, what do they represent